CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [2.1.1] - Unreleased
 
+### Changed
+-  Installer: Use init config.js from config.js.sample 
```

Comment:
Looks like linting is failing on this line due to extra spaces after the sentence.